src/ctap/ctap1.rs

```diff
@@ -745,12 +805,16 @@ mod test {
         let mut env = TestEnv::new();
         env.user_presence()
             .set(|_| panic!("Unexpected user presence check in CTAP1"));
-        let mut ctap_state = CtapState::new(&mut env, START_CLOCK_VALUE);
+        let mut ctap_state = CtapState::new(&mut env, CtapInstant::new(0_u64));
 
-        ctap_state.u2f_up_state.consume_up(START_CLOCK_VALUE);
-        ctap_state.u2f_up_state.grant_up(START_CLOCK_VALUE);
-        let response =
-            Ctap1Command::process_command(&mut env, &message, &mut ctap_state, TIMEOUT_CLOCK_VALUE);
+        ctap_state.u2f_up_state.consume_up(CtapInstant::new(0_u64));
+        ctap_state.u2f_up_state.grant_up(CtapInstant::new(0_u64));
+        let response = Ctap1Command::process_command(
+            &mut env,
+            &message,
+            &mut ctap_state,
+            CtapInstant::new((30001 * TEST_CLOCK_FREQUENCY_HZ as u64) / 1000),
```

Comment:
With embedded_time, is there a way to write this like:
`CtapInstant::new(0) + Milliseconds::new(30001)`
for easier readability?

Reply:
Done.